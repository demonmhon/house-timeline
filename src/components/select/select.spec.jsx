import React from 'react';
import { shallow } from 'enzyme';
import Select from './select';

describe('<Select />', () => {
  it('should match snapshot', () => {
    const items = ['one', 'two'];
    const wrapper = shallow(
      <Select label={'Select Component'} items={items} />
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('Should rendered with options properly', () => {
    const items = ['one', 'two'];
    const wrapper = shallow(
      <Select label={'Select Component'} items={items} />
    );
    expect(wrapper.find('option').length).toBe(items.length + 1);
  });
});
