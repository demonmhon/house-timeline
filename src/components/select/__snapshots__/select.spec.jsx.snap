// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Select /> should match snapshot 1`] = `
<label
  className="select"
  data-name="Select Component"
>
  <span
    className="select__label"
  >
    Select Component
  </span>
  <select
    onChange={[Function]}
    value=""
  >
    <option />
    <option
      key="one"
      value="one"
    >
      one
    </option>
    <option
      key="two"
      value="two"
    >
      two
    </option>
  </select>
</label>
`;
